build_library/sysext_mangle_flatcar-podman

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -euo pipefail
+rootfs="${1}"
+
+pushd "${rootfs}"
+
+rm -rf ./usr/{lib/debug/,lib64/cmake/,lib64/pkgconfig,include/,share/fish,share/aclocal,share/SLSA}
+
+mkdir -p ./usr/share/podman/etc
+cp -a ./etc/{fuse.conf,containers} ./usr/share/podman/etc/
+
+cat <<EOF >>./usr/lib/tmpfiles.d/podman.conf
+C /etc/containers - - - - /usr/share/podman/etc/containers
+C /etc/fuse.conf - - - - /usr/share/podman/etc/fuse.conf
+w /etc/subuid - - - - core:1065536:65536
+w /etc/subgid - - - - core:1065536:65536
+EOF
+
+mkdir -p ./usr/lib/systemd/network
+
+cat <<EOF >>./usr/lib/systemd/network/50-podman.network
+[Match]
+Type=bridge
+Name=podman*
+
+[Link]
+Unmanaged=yes
+EOF
+
+cat <<EOF >>./usr/lib/systemd/user/dbus.service
```

Comment:
The Podman extension isn't the best place for this, I think we should rather add this to the base image.

Reply:
Thanks for finding this. I think it's ok to enable it. The user session is also used by the systemd user instance.